File: app/src/main/java/com/karanchuk/roman/testtranslate/SettingsFragment.java
package com.karanchuk.roman.testtranslate;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

/**
 * Created by roman on 8.4.17.
 */

public class SettingsFragment extends Fragment {
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_settings, container, false);
        initToolbar();

        return view;
    }

    public void initToolbar(){
        ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
        if (actionBar != null) {
            actionBar.setShowHideAnimationEnabled(false);
            actionBar.setElevation(0);

            actionBar.setDisplayShowCustomEnabled(false);

            actionBar.setTitle("Settings");
            actionBar.setDisplayOptions(ActionBar.DISPLAY_SHOW_TITLE);

            actionBar.show();
        }
    }
}

File: app/src/main/java/com/karanchuk/roman/testtranslate/ui/settings/SettingsFragment.kt
package com.karanchuk.roman.testtranslate.ui.settings

import android.os.Bundle
import androidx.fragment.app.Fragment
import androidx.appcompat.app.ActionBar
import androidx.appcompat.app.AppCompatActivity
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.karanchuk.roman.testtranslate.R

/**
 * Created by roman on 8.4.17.
 */

class SettingsFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater,
                              container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_settings, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
//         initActionBar();
    }

    fun initActionBar() {
        val actionBar = (activity as AppCompatActivity).supportActionBar
        if (actionBar != null) {
            actionBar.setShowHideAnimationEnabled(false)
            actionBar.elevation = 0f
            actionBar.setDisplayShowCustomEnabled(false)
            actionBar.title = resources.getString(R.string.title_settings)
            actionBar.displayOptions = ActionBar.DISPLAY_SHOW_TITLE
            actionBar.show()
        }
    }
}
